	... languages oc ...
	... creating language pack: oc !
	... analyzing files ...
	    WARNING: Using autotextshare_en-US.zip instead of autotextshare_oc.zip
	    WARNING: Using palettes_en-US.zip instead of palettes_oc.zip
	    WARNING: Using tpllayoutimpr_en-US.zip instead of tpllayoutimpr_oc.zip
	    WARNING: Using tplpresntimpr_en-US.zip instead of tplpresntimpr_oc.zip
	    WARNING: Using tplwizagenda_en-US.zip instead of tplwizagenda_oc.zip
	    WARNING: Using tplwizdesktop_en-US.zip instead of tplwizdesktop_oc.zip
	    WARNING: Using tplwizfax_en-US.zip instead of tplwizfax_oc.zip
	    WARNING: Using tplwizletter_en-US.zip instead of tplwizletter_oc.zip
	    WARNING: Using tplwizreport_en-US.zip instead of tplwizreport_oc.zip
	    WARNING: Using tplwizstyles_en-US.zip instead of tplwizstyles_oc.zip
	    WARNING: Using wordbook_en-US.zip instead of wordbook_oc.zip
	... analyzing files with flag ARCHIVE ...

	**************************************************
	ERROR: ERROR: No packages defined!
diff --git a/patches/src680/apply b/patches/src680/apply
index 66b489d..ec5f68e 100644
--- a/patches/src680/apply
+++ b/patches/src680/apply
@@ -968,6 +968,7 @@ SectionOwner => davidf
 
 [ FrugalwareOnly ]
 gcc401-stlport45-include.diff, gghibo
+no-oc.diff, vmiklos
 
 
 # ( mmp@oo.o deals with defaults apparently )
diff --git a/patches/src680/no-oc.diff b/patches/src680/no-oc.diff
new file mode 100644
index 0000000..337310a
--- /dev/null
+++ b/patches/src680/no-oc.diff
@@ -0,0 +1,10 @@
+--- solenv/inc/postset.mk.orig	2008-02-13 13:40:48.000000000 +0100
++++ solenv/inc/postset.mk	2008-02-13 13:41:03.000000000 +0100
+@@ -39,6 +39,6 @@
+ defaultlangiso=en-US
+ 
+ # Complete list of all supported ISO codes
+-completelangiso=af ar as-IN be-BY bg br bn bn-BD bn-IN bs ca cs cy da de dz el en-GB en-US en-ZA eo es et eu fa fi fr ga gl gu-IN he hi-IN hr hu it ja ka km kn ko ku lo lt lv mk ms ml-IN mr-IN ne nb nl nn nr ns oc or-IN pa-IN pl pt pt-BR ru rw sk sl sh sr ss st sv sw sw-TZ te-IN ti-ER ta-IN th tn tr ts tg ur-IN uk uz ve vi xh zh-CN zh-TW zu
++completelangiso=af ar as-IN be-BY bg br bn bn-BD bn-IN bs ca cs cy da de dz el en-GB en-US en-ZA eo es et eu fa fi fr ga gl gu-IN he hi-IN hr hu it ja ka km kn ko ku lo lt lv mk ms ml-IN mr-IN ne nb nl nn nr ns or-IN pa-IN pl pt pt-BR ru rw sk sl sh sr ss st sv sw sw-TZ te-IN ti-ER ta-IN th tn tr ts tg ur-IN uk uz ve vi xh zh-CN zh-TW zu
+ 
+ alllangiso=$(strip $(defaultlangiso) $(subst,$(defaultlangiso), $(uniq $(subst,ALL,$(completelangiso) $(WITH_LANG)))))
